db/schema.rb

```diff
@@ -21,7 +21,7 @@
     t.string "age_range", default: "not_specified"
     t.boolean "has_alt_contact"
     t.string "voicemail_preference", default: "not_specified"
-    t.string "line", null: false
+    t.string "line_legacy"
```

Comment:
Intent here is to keep this around for a minute, kinda like we did for mongo_id, as a ref column for a little while until we can kill the column.